precompiles/macro/src/precompile/mod.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019-2022 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
```

Comment:
Do we need to update these licence headers? Keep it the same as other files.

Reply:
Good catch, thanks! Done!

Reply:
I'm fine with Wei's result but I'll get confirmation from the Moonbeam Foundation (PureStake Inc. will cease to exist at some point)

Reply:
I'm confirm it is fine